modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -174,6 +174,13 @@ image::dashboard-built-in-infrastructure.png[Example: Infrastructure Dashboard]
 The *Custom metrics* tab displays only if you have a Titanium subscription and select an app running on a supported version of Mule. See xref:anypoint-custom-metrics-connector.adoc[Anypoint Custom Metrics Connector].
 |===
 
+=== How Endpoint Use Metrics Are Gathered
+
+The built-in dashboards for apps contain charts that show endpoint metrics. The charts sort the available endpoints in a menu. When you designate a time period for the data that appears in the charts, the results shown in the endpoint charts can include multiple grouping intervals because the data is processed in intervals of time.
```

Comment:
```suggestion
The built-in dashboards for apps contain tables that show endpoint metrics. The tables sort the available endpoints in a menu. When you designate a time period for the data that appears in the tables, the results shown in the endpoint tables can include multiple grouping intervals because the data is processed in intervals of time.
```